recipes/x-transformers/recipe.yaml

```diff
@@ -0,0 +1,58 @@
+context:
+  version: "1.42.11"
+  python_min: 3.10
+
+package:
+  name: x-transformers
+  version: ${{ version }}
+
+source:
+  url: https://pypi.org/packages/source/x/x-transformers/x_transformers-${{ version }}.tar.gz
+  sha256: eb342ffc5b86193354f27dcdbfd3e13746dd102bf5ddffa21f3286fdc07bde0e
+
+build:
```

Comment:
I think this should be `noarch: python`. We ignore the issue with pytorch missing on windows given that this package does not actually have any platform dependencies. You can get rid of the `skip` section, and then use the `python_min` in the way described in https://conda-forge.org/docs/maintainer/knowledge_base/#noarch-python

Reply:
We just let it fail on Windows assuming the error is caused by the missing pytorch (which is the case here)